internal/services/environment_settings/datasource_environment_settings.go

```diff
@@ -147,6 +147,11 @@ func (d *EnvironmentSettingsDataSource) Schema(ctx context.Context, req datasour
 								MarkdownDescription: "Is read audit enabled",
 								Optional:            true,
 							},
+							"log_retention_period_in_days": schema.Int64Attribute{
+								Description:         "log_retention_period_in_days",
+								MarkdownDescription: "log_retention_period_in_days",
+								Optional:            true, Computed: true,
+							},
```

Comment:
why its computed?

Reply:
@eduardodfmex this is a good chance to better understand the fundamental concepts behind the schema behaviors.  Make sure that you take a few minutes to have a deeper understanding of optional, computed, etc. https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors

Reply:
OK, TNKS, based on my understanding this will be: Optional:            true.